scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/forms/DatasetEditForm.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.diging.scriptoocloud.web.forms;
+
+import edu.asu.diging.scriptoocloud.web.validation.FieldNotSame;
+
+import javax.validation.constraints.NotEmpty;
+
+@FieldNotSame(first = "oldName", second = "newName", message = "New Dataset name is the same as the old Dataset name")
```

Comment:
see my comment above (why?). Also, you would only need one form without this restriction.